manageiq-operator/pkg/helpers/miq-components/memcached.go

```diff
@@ -62,38 +62,33 @@ func NewMemcachedDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*
 	}
 	var repNum int32 = 1
 
-	spec := appsv1.DeploymentSpec{
-		Replicas: &repNum,
-		Selector: &metav1.LabelSelector{
-			MatchLabels: podLabels,
-		},
-		Template: corev1.PodTemplateSpec{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:   "memcached",
-				Labels: podLabels,
-			},
-			Spec: corev1.PodSpec{
-				Containers: []corev1.Container{container},
-			},
-		},
-	}
-
 	deployment := &appsv1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "memcached",
 			Namespace: cr.ObjectMeta.Namespace,
+			Labels:    podLabels,
+		},
+		Spec: appsv1.DeploymentSpec{
+			Replicas: &repNum,
```

Comment:
Putting this here prevents us from reconciling the replicas. I was thinking maybe we set the replicas, and template in the reconcile function rather than the entire deployment spec.